shopinvader/models/shopinvader_product.py

```diff
@@ -38,9 +38,9 @@ class ShopinvaderProduct(models.Model):
 
     @api.multi
     def _inverse_active(self):
-        inactive = self.filtered(lambda p: not p.active)
-        inactive.shopinvader_variant_ids.write(
-            {'active': False})
+        for inactive in self.filtered(lambda p: not p.active):
```

Comment:
@Cedric-Pigeon Why not `self.filtered(lambda p: not p.active).mapped('shopinvader_variant_ids').write( ....)`? To write all variants at once and reduce the number of sql queries